src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.test.tsx

```diff
@@ -318,24 +324,12 @@ describe('Financial Account Transactions', () => {
         ['8/8/2024', 'description2', 'category1Name', '15008', ''],
         ['8/7/2024', 'description3', 'category2Name', '', '36.2'],
       ];
-
-      const csvContent =
-        'data:text/csv;charset=utf-8,' +
-        csvContentArray
-          .map((row) =>
-            row
-              .map((field) => `"${String(field).replace(/"/g, '""')}"`)
-              .join(','),
-          )
-          .join('\n');
+      expect(buildURI).toHaveBeenCalledWith(csvContentArray, true);
 
       await waitFor(() => {
         expect(createElementSpy).toHaveBeenCalledWith('a');
         expect(appendChildSpy).toHaveBeenCalled();
-        expect(setAttributeSpy).toHaveBeenCalledWith(
-          'href',
-          encodeURI(csvContent),
-        );
+        expect(setAttributeSpy).toHaveBeenCalledWith('href', mockBlobUrl);
```

Comment:
Its a shame we are losing testing what the href will actually look like, but it makes sense to just mock the buildURI.